### DISCUSSION TITLE
With FormControl and zod validation

### DISCUSSION THREAD

**Discussion by KushalTanna24** [sersavan/shadcn-multi-select-component#22] [General]
Please show the example of how can i use the component inside Form control.?
As shadcn's FormField allows to use any component inside it to manage the state and error handling. Also is it possible we can see the integration of zod too.?

**Comment by sersavan** [comment]
An example of implementing a component inside a form with validation using zod can be found in this repository on page: [src/app/page.tsx](https://github.com/sersavan/shadcn-multi-select-component/blob/main/src/app/page.tsx)
